.github/actions/setup-gpg/action.yaml

```diff
@@ -0,0 +1,96 @@
+name: Setup GPG
```

Comment:
"technically" this is a "shared workflow" not an action, shared actions are a bit different? :thinking: Maybe this is splitting hairs and it's fine... probably? at least for now I don't think people will care or understand the difference.

Reply:
d'oh, you're right. I'll rename this.

Reply:
Actually, this _is_ an action, right?. [a 'composite action'](https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action). 
It has no `jobs:`, `on:`, or other things required to be a workflow.